docs/organizations/managing-security-and-risk.md

```diff
@@ -150,6 +150,21 @@ Codacy closes a finding in either of the following cases:
 !!! important
     Deleting a repository deletes all open findings belonging to that repository.
 
+### How Codacy manages findings detected during software composition analysis (SCA) {: id="opening-and-closing-sca-items"}
+
+!!! note
+    To make sure that Codacy detects dependency issues correctly:
+
+    -   [Enable code patterns](../repositories-configure/configuring-code-patterns.md) belonging to the Trivy tool. 
+
```

Comment:
since this is just one bullet point, I would change this to:
```suggestion
!!! note
    To make sure that Codacy detects dependency issues correctly, [enable code patterns](../repositories-configure/configuring-code-patterns.md) belonging to the Trivy tool. 

```